packages/web3/package.json

```diff
@@ -1,14 +1,13 @@
 {
     "name": "web3",
-    "version": "1.2.2",
+    "version": "1.2.3",
```

Comment:
Do all the other `package.json` get `"version"` bumped to `1.2.3` automatically?

Reply:
All package versions do get bumped to 1.2.3. But for the web3 package do we have to define the new version before the min file got bundled again otherwise will the ``Web3.version`` not be up to date.